sui/token_bridge/sources/objects/registered_tokens.move

```diff
@@ -75,6 +75,15 @@ module token_bridge::registered_tokens {
         !is_wrapped<C>(self)
     }
 
+    public(friend) fun treasury_cap<C>(
```

Comment:
should `C` be `CoinType`?

Reply:
we've been using C for brevity in some places